src/index.js

```diff
@@ -0,0 +1,130 @@
+import _, { add, times } from 'lodash';
+import { clearAll } from './script.js';
+import taskCompleted from './script.js';
+import './style.css';
+
+
+
+const formContainer = document.getElementById('form')
+const inputElement = document.getElementById('input-element');
+const myContainer = document.getElementById('items-container');
+const clearSelectedList = document.getElementById('footer-a')
+
+
+
```

Comment:
The `index.js` file should have an array of some simple *to do tasks* (array of objects) with each object containing three keys - description \[`string`\], completed \[`bool`\], index: \[`number`\]. 

For example:
```
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
```

- Kindly make sure that you have a few sample tasks in the array (at least 3 preferably) and they must be displayed in ascending order of the index. Currently, there are no sample tasks on the page.

![image](https://user-images.githubusercontent.com/38088818/194694874-e7d78080-9027-4640-87ef-2cde63683c36.png)

Reply:
Fixed.